File: src/main/java/me/racci/sylphia/origins/objects/OriginAttribute.java
package me.racci.sylphia.origins.objects;

import org.bukkit.attribute.Attribute;

public class OriginAttribute {
	private Attribute attribute;
	private double value;

	public OriginAttribute(Attribute attribute, double value) {
		this.attribute = attribute;
		this.value = value;
	}

	public Attribute getAttribute() {
		return this.attribute;
	}

	public void setAttribute(Attribute attribute) {
		this.attribute = attribute;
	}

	public double getValue() {
		return this.value;
	}

	public void setValue(double value) {
		this.value = value;
	}

}

File: src/main/kotlin/me/racci/sylphia/origins/OriginAttribute.kt
@file:Suppress("unused")
@file:JvmName("OriginAttribute")
package me.racci.sylphia.origins

import org.bukkit.attribute.Attribute
import org.bukkit.attribute.AttributeModifier

class OriginAttribute(var attribute: Attribute, var value: Double)

data class BaseAttribute(var attribute: Attribute, var double: Double)

data class AttributeModifier(var attribute: Attribute, var modifier: AttributeModifier)